review.md

```diff
@@ -4,11 +4,11 @@ This is a very interesting paper that researched the Deep-Q based DRL agents for
 1) Some of the agents names need to be mentioned fully: like DRL what is its full name? list all similar names in the text. You may use notation table for all of them for the readers benefit.
 2) Page 6 - Line 16: what is the usage of serialize and deserialize.
```

Comment:
> Page 6 - Line 16: what is the usage of serialize and deserialize.

I feel it is already explained so I'm not sure how to improve it. Should we quote Gym documentation for that ?

Our sentence is _"• Gym contains utilities in order to serialize and deserialize the Observation and Action concepts from a human-readable object to a vector suitable for a neural network."_